solution/selection.py

```diff
@@ -1,6 +1,22 @@
-# Selects 2 parents from the current population
-def parent_selection(population):
-    pass
+import random 
+
+
+def fitness(env,x):
+    f,p,e,t = env.play(pcont=x)
+    return f
+
+def tournament(population, k, env):
+    selected_individuals = random.sample(population, k)
+    # Find the best individual among the selected based on fitness
+    best_individual = max(selected_individuals, key=lambda ind: fitness(env, ind['pcont']))
```

Comment:
what's the `ind['pcont']` that you're passing to the fitness function?

Reply:
It refers to the control parameters that are used during the evaluation of fitness by env.play(pcont=x), and ind is the individual. But I think actually I should change it, coz our individual is a list, right?

Reply:
I don't know what parameter their function expects. Anyway, we can merge this for now and handle it later.